tpm/tpm.go

```diff
@@ -1005,3 +1005,16 @@ func PcrReset(rw io.ReadWriter, pcrs []int) error {
 	}
 	return nil
 }
+
+// ForceClear is normally used by firmware but on some platforms
```

Comment:
In addition to the explanation of why this command is useful to have in this library, please mention briefly the expected effect of calling this function (for completeness, since the caller can probably guess from the name).

Reply:
Done